docs/tutorials/dev-setup.rst

```diff
@@ -0,0 +1,44 @@
+Development setup
```

Comment:
This is duplicated. Installation instructions also exist in the pyserver docs. Maybe have only one place to explain installation and reference that. Just wanted you to have this in mind, doesn't need to be changed right now.

Reply:
You're right, the Idea was to move all installation and setup instructions to the `Development setup` section and reference it if needed.
I'm aware of that problem and will fix it in the next version.